### ISSUE TITLE
ToolClasspathSuite failing in Scala community build

### ISSUE THREAD

**Issue by SethTisue** [scalacenter/scalafix#896]
seen at https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3583/consoleFull

```
[scalafix] [info] ToolClasspathSuite:
[scalafix] [info] - --tool-classpath is respected when compiling from source *** FAILED *** (1 second, 733 milliseconds)
[scalafix] [info]   java.util.NoSuchElementException: 2 errors
[scalafix] [info] [E0] /tmp/scalafix2232454107816729716FormatRule.scala:1:11 error: object scalafmt is not a member of package org
[scalafix] [info] import org.scalafmt._
[scalafix] [info]            ^
[scalafix] [info] 
[scalafix] [info] [E1] /tmp/scalafix2232454107816729716FormatRule.scala:7:20 error: not found: value Scalafmt
[scalafix] [info]     val formatted = Scalafmt.format(ctx.tokens.mkString).get
[scalafix] [info]                     ^
[scalafix] [info]   at metaconfig.Configured.get(Configured.scala:11)
[scalafix] [info]   at scalafix.tests.reflect.ToolClasspathSuite.$anonfun$new$1(ToolClasspathSuite.scala:51)
[scalafix] [info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[scalafix] [info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[scalafix] [info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[scalafix] [info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[scalafix] [info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[scalafix] [info]   at org.scalatest.FunSuiteLike$$anon$1.apply(FunSuiteLike.scala:186)
[scalafix] [info]   at org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
[scalafix] [info]   at org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
[scalafix] [info]   ...
```

**Comment by SethTisue** [comment]
excluded for now: scala/community-builds@5b0a2d4af545af0c0d337934c274a7c7059f1a74

**Comment by SethTisue** [comment]
For the record:

The test remains excluded in the 2.12 community build, but I don't care if y'all don't; the 2.12 build is mostly frozen these days anyway.

In the 2.13 build, we aren't running scalafix's test suite at all, as per https://github.com/scala/community-build/pull/1290 . That's unfortunate I would be interested in participating in doing something about it, but I can't afford to get too involved, either.

In short, yes it makes sense to close this ticket :-)
